server/database/dbOperations.ts

```diff
@@ -229,3 +239,397 @@ export const removeTableFromConfig = async (
     throw error;
   }
 };
+
+// Annotated Collections Operations
+
+export const createAnnotatedCollection = async (
+  db: DatabaseConnection,
+  collection: Omit<AnnotatedCollection, "id" | "created_at" | "updated_at">,
+  incidentData?: Omit<Incident, "collection_id">,
+  entries?: Array<{
+    source_table: string;
+    source_id: string;
+    notes?: string;
+  }>,
+): Promise<AnnotatedCollection> => {
+  const pgClient = db as Client;
+
+  try {
+    await pgClient.query("BEGIN");
+
+    // Create the main collection
+    const collectionQuery = `
+      INSERT INTO annotated_collections (name, description, collection_type, status, created_by, is_active, metadata)
+      VALUES ($1, $2, $3, $4, $5, $6, $7)
+      RETURNING *
+    `;
+
+    const collectionResult = await pgClient.query(collectionQuery, [
+      collection.name,
+      collection.description,
+      collection.collection_type,
+      collection.status,
+      collection.created_by,
+      collection.is_active,
```

Comment:
```suggestion
      collection.created_by,
```

The other two were removed from annotated collections in the last PR.